app/controllers/organizations_controller.rb

```diff
@@ -125,6 +126,17 @@ def set_users_github_organizations
   end
   # rubocop:enable AbcSize
 
+  # Check if the current user has any organizations with admin privilege, if so add the user to the corresponding
+  # classroom automatically.
+  def auto_scan_organizations_access
+    @users_github_organizations.each do |organization|
+      next unless organization[:classroom].present?
+      next if organization[:classroom].users.include?(current_user) || current_user.staff?
```

Comment:
Why would we want to skip over the user if they are a staff member? Staff members can be part of the organization.